install_files/ansible-base/securedrop-configure.yml

```diff
@@ -60,6 +60,7 @@
 
     - name: securedrop_app_gpg_fingerprint
       prompt: Full fingerprint for the SecureDrop Application GPG Key
+      default: ' '
```

Comment:
The empty string handling was causing skippage due to the conditional on the vars-writing task:

```
when: item.var_value is defined and item.var_value != ""
```

That shouldn't be necessary, given the `-f` flag, but will drill down on QA when resolving #1857.